board.html

```diff
@@ -33,6 +33,20 @@ <h3>Add a new contact</h3>
     <div id="Search" class="tabcontent">
         <h3>Search contacts</h3>
         <input type="text" class="inputClear" id = "searchText" placeholder="Search Here" /><br />
+        
+        <span>Search by:</span></br>
+        <input type="checkbox" class="smallbox" name="searchFname" id="searchFname" checked>
```

Comment:
please use snake_case instead of camel 
and don't do Fname, use something like ```search-first-name``` instead